src/data-accessor.ts

```diff
@@ -39,6 +40,17 @@ function parseCsvToPower(csv: CsvFile): Power[] {
   });
 }
 
+function parseAttributesImpacted(rawAttributesEntry: string): AttributeImpact[] {
+  return rawAttributesEntry.split(", ").map((rawAttribute) => {
+    const attrArr = rawAttribute.split(" ");
+    return {
+      attribute: parseAttribute(attrArr[0]),
+      minImpact: parseInt(attrArr[1]),
+      maxImpact: parseInt(attrArr[attrArr.length - 1]),
```

Comment:
it may not be an issue here, but parseInt takes a second parameter for the base (ie 10 for decimal), and convention is to always pass the base to avoid blunders like this https://dev.to/lucis/why-you-should-never-map-parseint-49nj

ideally we would have a lint rule or it.